src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -114,11 +130,59 @@ function CustomizedInputBase(props) {
           alignItems="center"
           justify="flex-start"
         >
-          <Grid item xs={12}>
+          <Grid item>
             <Typography variant="h1" className={classes.h1}>
               Search
             </Typography>
           </Grid>
+          <HtmlTooltip
```

Comment:
Let's break this out into its own component, `SearchHelp.jsx` so it's not so massive in this file.  You should be able to use it here with `<SearchHelp />`